packages/vincent-tools/vincent-tool-erc20-approval/src/lib/vincent-tool-wrapped.ts

```diff
@@ -0,0 +1,23 @@
+import { vincentToolHandler } from '@lit-protocol/vincent-tool-sdk';
+import { z } from 'zod';
+
+import { Erc20ApprovalToolDef, Erc20ApprovalToolParamsSchema } from './vincent-tool';
```

Comment:
Just Erc20ApprovalTool, since the def is what's passed in to `createVincentTool` :)